src/components/UnstyledLink/tests/UnstyledLink.test.tsx

```diff
@@ -57,5 +57,17 @@ describe('<UnstyledLink />', () => {
       ).find('a');
       expect(anchorElement.prop('download')).toBeFalsy();
     });
+
+    describe('accessibilityLabel', () => {
+      it('passes prop', () => {
+        const mockAccessibilityLabel = 'mock accessibility label';
+        const anchorElement = mountWithAppProvider(
```

Comment:
Similarly, we'll want to use mountWithApp, and do the same changes that was suggested for Link.test.tsx.